src/app/modules/main/profile_section/contacts/controller.nim

```diff
@@ -73,6 +73,10 @@ proc init*(self: Controller) =
     var args = ContactArgs(e)
     self.delegate.onVerificationRequestDeclined(args.contactId)
 
+  self.events.on(SIGNAL_CONTACT_VERIFICATION_CANCELLED) do(e: Args):
```

Comment:
Imho `SIGNAL_CONTACT_VERIFICATION_DECLINED` and `SIGNAL_CONTACT_VERIFICATION_CANCELLED` both can be deleted because of similar behaviour in `onVerificationRequestUpdatedOrAdded`